packages/core/src/command/ai-brief/create.ts

```diff
@@ -185,7 +190,9 @@ export async function summarizeFilteredBundleWithAI(
 
     const duration = elapsedTimeFromNow(startedAt);
     log(
-      `Done. Finished in ${duration} ms. Total tokens used: ${totalTokensUsed}. Input cost: ${costs.input}, output cost: ${costs.output}. Total cost: ${costs.total}`
+      `Done. Finished in ${duration} ms. Total tokens used: ${JSON.stringify(
+        totalTokensUsed
```

Comment:
We're not sending this to analytics, right? I think its an important metric.

Reply:
Added!